crux-agent/app/core/providers/lmstudio.py

```
@@ -0,0 +1,624 @@
+                base_url=base_url,
+                timeout=timeout,
+                max_retries=max_retries,
+            )
+            logger.debug("Using official LMStudio SDK")
+        except ImportError:
+            logger.debug("LMStudio SDK not available, using HTTP client")
+    
+    def _with_json_retry(self, fn, *args, **kwargs):
+        """
+        Helper method to perform retries for JSON parsing using tenacity.
+        
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)